app/views/communities/_form.html.erb

```diff
@@ -4,15 +4,17 @@
     <div class="row">
       <div class="col-sm-12">
         <div class="create-post m-3">
-          <div class="form-group"> 
+          <div class="form-group">
             <%= label_tag :profile_picture %>
-            <%= form.file_field :profile_image, class: "form-control" %>
+            <%= form.file_field :profile_image, class: "form-control "+is_error(@community,(:profile_image)).to_s %>
```

Comment:
can you move the error part to a helper?

Reply:
I was thinking we can move the "form-control" and .to_s part also into the helper method to make this look better. What do you think?